src/frontend/utils/telemetry/FrontendTracer.ts

```diff
@@ -43,6 +43,9 @@ const FrontendTracer = async (collectorString: string) => {
         '@opentelemetry/instrumentation-fetch': {
           propagateTraceHeaderCorsUrls: /.*/,
           clearTimingResources: true,
+          applyCustomAttributesOnSpan(span) {
```

Comment:
This unique way to add attributes on spans at the instrumentation initialization needs to be documented in the [frontend docs for browser instrumentation](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/services/frontend.md#browser-instrumentation)

Reply:
Done 😄